### PR TITLE
New images for November 2024 Spotlight

### PR DESCRIPTION
**Pull request by amada-ech**
here they are!

## Checklist
- [ x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [x ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).

### CODE REVIEW

**Comment by amada-ech** [comment]
I just realized I maybe shouldnt have titled this "New images for November 2024 Spotlight"... if the spotlight entries themselves will fall under the same PR, then this project is more than just images, its the 8 spotlight entries themselves too FYI!

File: website/docs/community/spotlight/jenna-jordan.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+id: jenna-jordan
+title: Jenna Jordan
+description: |
+  I am a Senior Data Management Consultant with Analytics8, where I advise clients on dbt best practices (especially regarding dbt Mesh and the various shifts in governance and strategy that come with it). My experiences working within a dbt Mesh architecture and all of the difficulties organizations could run into with such a major paradigm shift inspired my peer exchange (role-playing/simulation game) at <a href="https://coalesce.getdbt.com/agenda" rel="noopener noreferrer" target="_blank">Coalesce 2024</a>: “Governance co-lab: We the people, in order to govern data, do establish processes.” I also experimented with bringing role-playing scenarios to data problems at the September 2024 <a href="https://www.meetup.com/chicago-dbt-meetup/" rel="noopener noreferrer" target="_blank">Chicago dbt Meetup</a>, hosted by Analytics8. I occasionally write long blog posts on my website, if you're up for the read.
```
Comment (matthewshaver) [inline website/docs/community/spotlight/jenna-jordan.md]: Syntax might be breaking Vercel build
```suggestion
  I am a Senior Data Management Consultant with Analytics8, where I advise clients on dbt best practices (especially regarding dbt Mesh and the various shifts in governance and strategy that come with it). My experiences working within a dbt Mesh architecture and all of the difficulties organizations could run into with such a major paradigm shift inspired my peer exchange (role-playing/simulation game) at <a href="https://coalesce.getdbt.com/agenda" rel="noopener noreferrer" target="_blank">Coalesce 2024</a>: Governance co-lab: We the people, in order to govern data, do establish processes. I also experimented with bringing role-playing scenarios to data problems at the September 2024 <a href="https://www.meetup.com/chicago-dbt-meetup/" rel="noopener noreferrer" target="_blank">Chicago dbt Meetup</a>, hosted by Analytics8. I occasionally write long blog posts on my website, if you're up for the read.
```

File: website/docs/community/spotlight/jenna-jordan.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+id: jenna-jordan
+title: Jenna Jordan
+description: |
+  I am a Senior Data Management Consultant with Analytics8, where I advise clients on dbt best practices (especially regarding dbt Mesh and the various shifts in governance and strategy that come with it). My experiences working within a dbt Mesh architecture and all of the difficulties organizations could run into with such a major paradigm shift inspired my peer exchange (role-playing/simulation game) at <a href="https://coalesce.getdbt.com/agenda" rel="noopener noreferrer" target="_blank">Coalesce 2024</a>: Governance co-lab: We the people, in order to govern data, do establish processes. I also experimented with bringing role-playing scenarios to data problems at the September 2024 <a href="https://www.meetup.com/chicago-dbt-meetup/" rel="noopener noreferrer" target="_blank">Chicago dbt Meetup</a>, hosted by Analytics8. I occasionally write long blog posts on my website, if you're up for the read.
```
Comment (matthewshaver) [inline website/docs/community/spotlight/jenna-jordan.md]: Wasn't the problem
```suggestion
  I am a Senior Data Management Consultant with Analytics8, where I advise clients on dbt best practices (especially regarding dbt Mesh and the various shifts in governance and strategy that come with it). My experiences working within a dbt Mesh architecture and all of the difficulties organizations could run into with such a major paradigm shift inspired my peer exchange (role-playing/simulation game) at <a href="https://coalesce.getdbt.com/agenda" rel="noopener noreferrer" target="_blank">Coalesce 2024</a>: "Governance co-lab: We the people, in order to govern data, do establish processes." I also experimented with bringing role-playing scenarios to data problems at the September 2024 <a href="https://www.meetup.com/chicago-dbt-meetup/" rel="noopener noreferrer" target="_blank">Chicago dbt Meetup</a>, hosted by Analytics8. I occasionally write long blog posts on my website, if you're up for the read.
```

File: website/docs/community/spotlight/jenna-jordan.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
... February 2022 – less than a month after starting as a data engineer with the City of Boston Analytics Team. About a year later, I spearheaded the adoption of dbt at the City and got to build the project and associated architecture from scratch – which is probably the best learning experience you could ask for! I saw the value dbt could bring to improving data management processes at the City, and I knew there were other cities and local governments that could benefit from dbt as well, which motivated me to find my fellow co-speakers Ian Rose and Laurie Merrell to give a talk at Coalesce 2023 called <a href="https://www.youtube.com/watch?v=6aX7tAfMmIM&“ rel="noopener noreferrer" target="_blank">“From Coast to Coast: Implementing dbt in the public sector”.</a> As a part of our goal to identify and cultivate a community of dbt practitioners in the public (and adjacent) sectors, we also started the dbt Community Slack channel <a href="https://getdbt.slack.com/archives/C05MNU6QB5L/“ rel="noopener noreferrer" target="_blank">#industry-public-sector</a>. That experience allowed me to continue to grow my career and find my current role - as well as connect with so many amazing data folks!
```
Comment (matthewshaver) [inline website/docs/community/spotlight/jenna-jordan.md]:
```suggestion
My dbt learning journey kicked off with the CoRise (now Uplimit) course <a href="https://uplimit.com/course/analytics-engineering-with-dbt/" rel="noopener noreferrer" target="_blank">Analytics Engineering with dbt</a>, with Emily Hawkins and Jake Hannan, in February 2022 – less than a month after starting as a data engineer with the City of Boston Analytics Team. About a year later, I spearheaded the adoption of dbt at the City and got to build the project and associated architecture from scratch – which is probably the best learning experience you could ask for! I saw the value dbt could bring to improving data management processes at the City, and I knew there were other cities and local governments that could benefit from dbt as well, which motivated me to find my fellow co-speakers Ian Rose and Laurie Merrell to give a talk at Coalesce 2023 called <a href="https://www.youtube.com/watch?v=6aX7tAfMmIM&“ rel="noopener noreferrer" target="_blank">“From Coast to Coast: Implementing dbt in the public sector”.</a> As a part of our goal to identify and cultivate a community of dbt practitioners in the public (and adjacent) sectors, we also started the dbt Community Slack channel <a href="https://getdbt.slack.com/archives/C05MNU6QB5L/“ rel="noopener noreferrer" target="_blank">#industry-public-sector</a>. That experience allowed me to continue to grow my career and find my current role - as well as connect with so many amazing data folks!
```

File: website/docs/community/spotlight/christophe-oudoar.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+pronouns: he/him
+location: Montpellier, France
+jobTitle: Staff Engineer
+companyName: Teads
+socialLinks:
+  - name: X
+    link: https://x.com/Kayrnt
+  - name: LinkedIn
+    link: https://www.linkedin.com/in/christopheoudar/
+ - name: Substack
```
Comment (matthewshaver) [inline website/docs/community/spotlight/christophe-oudoar.md]:
```suggestion
  - name: Substack
```

File: website/docs/community/spotlight/jenna-jordan.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
```
Comment (matthewshaver) [inline website/docs/community/spotlight/jenna-jordan.md]:
```suggestion
My dbt learning journey kicked off with the CoRise (now Uplimit) course <a href="https://uplimit.com/course/analytics-engineering-with-dbt/" rel="noopener noreferrer" target="_blank">Analytics Engineering with dbt</a>, with Emily Hawkins and Jake Hannan, in February 2022 – less than a month after starting as a data engineer with the City of Boston Analytics Team. About a year later, I spearheaded the adoption of dbt at the City and got to build the project and associated architecture from scratch – which is probably the best learning experience you could ask for! I saw the value dbt could bring to improving data management processes at the City, and I knew there were other cities and local governments that could benefit from dbt as well, which motivated me to find my fellow co-speakers Ian Rose and Laurie Merrell to give a talk at Coalesce 2023 called <a href="https://www.youtube.com/watch?v=6aX7tAfMmIM&" rel="noopener noreferrer" target="_blank">“From Coast to Coast: Implementing dbt in the public sector”.</a> As a part of our goal to identify and cultivate a community of dbt practitioners in the public (and adjacent) sectors, we also started the dbt Community Slack channel <a href="https://getdbt.slack.com/archives/C05MNU6QB5L/“ rel="noopener noreferrer" target="_blank">#industry-public-sector</a>. That experience allowed me to continue to grow my career and find my current role - as well as connect with so many amazing data folks!
```

File: website/docs/community/spotlight/jenna-jordan.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
... February 2022 – less than a month after starting as a data engineer with the City of Boston Analytics Team. About a year later, I spearheaded the adoption of dbt at the City and got to build the project and associated architecture from scratch – which is probably the best learning experience you could ask for! I saw the value dbt could bring to improving data management processes at the City, and I knew there were other cities and local governments that could benefit from dbt as well, which motivated me to find my fellow co-speakers Ian Rose and Laurie Merrell to give a talk at Coalesce 2023 called <a href="https://www.youtube.com/watch?v=6aX7tAfMmIM&" rel="noopener noreferrer" target="_blank">“From Coast to Coast: Implementing dbt in the public sector”.</a> As a part of our goal to identify and cultivate a community of dbt practitioners in the public (and adjacent) sectors, we also started the dbt Community Slack channel <a href="https://getdbt.slack.com/archives/C05MNU6QB5L/“ rel="noopener noreferrer" target="_blank">#industry-public-sector</a>. That experience allowed me to continue to grow my career and find my current role - as well as connect with so many amazing data folks!
```
Comment (matthewshaver) [inline website/docs/community/spotlight/jenna-jordan.md]:
```suggestion
My dbt learning journey kicked off with the CoRise (now Uplimit) course <a href="https://uplimit.com/course/analytics-engineering-with-dbt/" rel="noopener noreferrer" target="_blank">Analytics Engineering with dbt</a>, with Emily Hawkins and Jake Hannan, in February 2022 – less than a month after starting as a data engineer with the City of Boston Analytics Team. About a year later, I spearheaded the adoption of dbt at the City and got to build the project and associated architecture from scratch – which is probably the best learning experience you could ask for! I saw the value dbt could bring to improving data management processes at the City, and I knew there were other cities and local governments that could benefit from dbt as well, which motivated me to find my fellow co-speakers Ian Rose and Laurie Merrell to give a talk at Coalesce 2023 called <a href="https://www.youtube.com/watch?v=6aX7tAfMmIM&" rel="noopener noreferrer" target="_blank">"From Coast to Coast: Implementing dbt in the public sector".</a> As a part of our goal to identify and cultivate a community of dbt practitioners in the public (and adjacent) sectors, we also started the dbt Community Slack channel <a href="https://getdbt.slack.com/archives/C05MNU6QB5L/" rel="noopener noreferrer" target="_blank">#industry-public-sector</a>. That experience allowed me to continue to grow my career and find my current role - as well as connect with so many amazing data folks!
```

**Review by matthewshaver — APPROVED** [review]

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**